### DISCUSSION TITLE
상태관리 라이브러리 선정

### DISCUSSION THREAD

**Discussion by SeungrokYoon** [wanted-pre-onboarding-team-12th-7/pre-onboarding-12th-2-7#1] [General]
어떤 상태관리 라이브러리를 사용해보셨나요?

혹시 팀 과제에서 사용하고 싶은 상태관리 라이브러리가 있으신가요?

Recoil 공식문서: https://recoiljs.org/ko/
Jotai 공식문서: https://jotai.org/

Redux-toolkit 공식문서: https://redux-toolkit.js.org/tutorials/quick-start 
Zustand 공식문서: https://zustand-demo.pmnd.rs/

**Comment by SeungrokYoon** [comment]
저는 만약 필요하다면 Redux Toolkit을 활용해서 개인과제를 진행하려 합니다!

**Reply by SeungrokYoon** [reply]
Redux Toolkit으로 구현하고, Recoil로 변경해보는 것도 재밌겠어요 ㅎㅎ

**Comment by 5unk3n** [comment]
저는 Recoil만 사용해봤습니다. 아까 zustand를 많이 말씀해주셔서 팀 과제에서 사용하게 된다면 공부하면서 해보겠습니다!ㅎㅎ

**Reply by SeungrokYoon** [reply]
저는 Zustand 는 Redux의 미니버전으로, Jotai는 Recoil의 미니버전 느낌이 나더라구요ㅎㅎ 

시간이 괜찮으시다면 Zustand를 개인플젝에 부분적으로 도입해보면서 익히는 경험도 좋을 것 같아요!

**Comment by wisdomin121** [comment]
저는 zustand만 사용해봤어요 !

만약 팀 과제에서 사용할 거라면 zustand도 좋고, 제일 많이 사용하고 있는 redux도 한 번 사용해보고 싶은 마음이 있습니다.

**Reply by SeungrokYoon** [reply]
오옹... zustand를 사용해보셨군요, 사용법이 생각보다 간단해서 redux보다 러닝커브가 그렇게 높지 않다 생각해요 보일러 플레이트 코드도 상대적으로 적구요ㅎㅎ 
민지님두 시간이 괜찮으시다면 redux-toolkit 을 개인플젝에 적용할 부분이 있다면 먼저 적용해보시는 것도 좋아보여요! 

저는 Redux-toolkit을 주로 사용했었는데, 이번에 시간이 괜찮다면 Recoil을 사용해보려구요!

**Comment by salmontaker** [comment]
둘 다 사용해보지는 않았지만, 러닝커브가 낮은 쪽을 사용해보았으면 합니다!
개인 프로젝트는 contextAPI를 사용해서 구현했는데, 결정된 쪽으로 리팩토링 해보고 싶네요!
